x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/agent_explorer/agent_explorer.spec.ts

```diff
@@ -42,7 +43,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     });
   }
 
-  registry.when('Agent explorer when data is not loaded', { config: 'basic', archives: [] }, () => {
+  registry.when('Agent explorer when data is not loaded', () => {
```

Comment:
Q: since some test still require passing `archives`, how we could config that in the new agnostic approach?

Reply:
Good question! I spent some time thinking about that, but decided not to implement anything at service level to provide the loading of the es_archives, at least for this PR.

I tried to make `registry` as generic as possible because that are other apps using a similar implementation. We could revisit this service and add the es_archives loading, without strongly typing `archives` with APM-specific archives.